### PR TITLE
Added the recommended ruleset of ESlint, RuboCop, PMD

### PR DESCRIPTION
**Pull request by sleeeto**
Add Sider's recommended ruleset documents of ESLint, RuboCop, PMD.

- Reference
  - https://github.com/sider/sideci/issues/9963
  - https://github.com/sider/sideci/issues/9964
  - https://github.com/sider/sideci/issues/10434

### CODE REVIEW

File: docs/getting-started/recommended-rules.md [hunk 1]
```diff
@@ -26,3 +26,6 @@ Sider applies the recommended ruleset to the analysis tools below. Please follow
 - [cpplint](../tools/cplusplus/cpplint.md#recommended-ruleset)
 - [Flake8](../tools/python/flake8.md#default-configuration-for-flake8)
 - [PHP_CodeSniffer](../tools/php/code-sniffer.md#default-configuration)
+- [ESLint](../tools/javascript/eslint.md#default-configuration-for-eslint)
+- [RuboCop](../tools/ruby/rubocop.md#default-configuration-for-rubocop)
+- [PMD](../tools/java/pmd.md#default-configuration-for-pmd)
```
Comment (ybiquitous) [inline docs/getting-started/recommended-rules.md:31]: Please sort alphabetically

**Comment by ybiquitous** [comment]
Wait for the next release of Runners.

**Review by suetsuguk — APPROVED** [review]
There is nothing from me except Koba-san's request.

**Review by ybiquitous — APPROVED** [review]

**Comment by ybiquitous** [comment]
Again, please wait for the merge until the next release.

**Comment by sleeeto** [comment]
OK, thanks.

File: docs/getting-started/recommended-rules.md (hunk 1 above)
Comment (sleeeto) [reply docs/getting-started/recommended-rules.md]: OK, Updated it.

File: docs/news/2021.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ This document describes notable changes on Sider in the year 2021.
 
 ---
 
+## Provide new recommended rulesets
+
+<time class="news-date" datetime="2021-05-18">May 18, 2021</time>
+
+Today, we have just released our new recommended rulesets for [ESLint](../tools/javascript/eslint.md), [PMD](../tools/java/pmd.md), and [RuboCop](../tools/ruby/rubocop.md).
+Please check out [Recommended Ruleset](../getting-started/recommended-rules.md) for more details about the rulesets.
+
```
Comment (suetsuguk) [inline docs/news/2021.md:20]: Add a section separator with `---`.

```suggestion

---

```

Comment (sleeeto) [reply docs/news/2021.md:20]: OK, fixed.

**Comment by sleeeto** [comment]
@ybiquitous Ready.

**Comment by suetsuguk** [comment]
I have verified your changes. Thank you.

**Pull request merged**